website/docs/r/iploadbalancing_http_frontend.html.markdown

```diff
@@ -51,6 +51,7 @@ The following arguments are supported:
 * `disabled` - Disable your frontend. Default: 'false'
 * `ssl` - SSL deciphering. Default: 'false'
 * `redirect_location` - Redirection HTTP'
+* `http_header` - HTTP headers to add to the frontend. List of string.
 
 ## Attributes Reference
```

Comment:
Moreover, is it possible to add missing attributes in the list please.

`port`, `service_name` and `zone` seems to be missing.